Source/WebGPU/WGSL/Metal/MetalFunctionWriter.cpp

```diff
@@ -66,6 +66,21 @@ namespace Metal {
 #define DEFINE_BOUND_HELPER(__name, __capitalizedName, __lowerBound, __upperBound, ...) \
     DEFINE_BOUND_HELPER_RENAMED(__name, __capitalizedName, __name, __lowerBound, __upperBound, __VA_ARGS__)
 
+#define DEFINE_VOLATILE_BOUND_HELPER_RENAMED(__name, __capitalizedName, __metalFunction, __lowerBound, __upperBound, ...) \
```

Comment:
not sure if there was a better way to make this macro @tadeuzagallo without largely copying the non-volatile version

But it is fairly simple, so avoiding the duplication is probably more effort than its worth